src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -100,4 +100,7 @@
   <data name="InvalidOperation_WrongOidInAsnCollection" xml:space="preserve">
     <value>AsnEncodedData element in the collection has wrong Oid value: expected = '{0}', actual = '{1}'.</value>
   </data>
+  <data name="PlatformNotSupported_CryptographyPkcs" xml:space="preserve">
+    <value>System.Security.Cryptography.Pkcs is not yet supported on this platform</value>
```

Comment:
About the namespace, @bartonjs suggested not using an expanded acronym since we support PKCS#1 (RSA) and PKCS#12 (PFX), and PKCS#7 data export, but not PKCS#7 SignedCms or EnvelopedCms.